src/psyclone/transformations.py

```diff
@@ -2618,15 +2618,7 @@ class ACCKernelsTrans(RegionTrans):
 
     '''
     excluded_node_types = (CodeBlock, Return, PSyDataNode,
-                           psyGen.HaloExchange)
-
-    @property
-    def name(self):
-        '''
-        :returns: the name of this transformation class.
-        :rtype: str
-        '''
-        return "ACCKernelsTrans"
+                           psyGen.HaloExchange, WhileLoop)
```

Comment:
And similar to the previous comment, since we are here, could we bring this transformation under its own file in psyir/transformations? Also remove the remaining backticks in the apply docstring.

Reply:
Why not? Done. (Assuming you meant backslashes rather thank backticks?)